NDB.Covid19/NDB.Covid19.iOS/Views/FarwellSmittestopPage/NotActiveViewController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using Foundation;
+using I18NPortable;
+using NDB.Covid19.iOS.Utils;
+using NDB.Covid19.PersistedData;
+using NDB.Covid19.ViewModels;
+using UIKit;
+
+namespace NDB.Covid19.iOS.Views.NotActive
+{
+    public partial class NotActiveViewController : BaseViewController
+    {
+        public static DialogViewModel LanguageChangedDialog => new DialogViewModel
+        {
+            Title = "SETTINGS_GENERAL_CHOOSE_LANGUAGE_HEADER".Translate(),
+            Body = "SETTINGS_GENERAL_RESTART_REQUIRED_TEXT".Translate(),
+            OkBtnTxt = "SETTINGS_GENERAL_DIALOG_OK".Translate()
+        };
+        private bool _languageChanged = false;
+
+        public NotActiveViewController(IntPtr handle) : base(handle)
+        {
+        }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            SetupLayout();
+
+            // TODO: Check if any consent accepted, if yes -> Go to farawell
+        }
+
+        public void SetupLayout()
+        {
+            StyleUtil.InitLabel(descriptionLabel, StyleUtil.FontType.FontBold,
+                SmittestopNotActiveViewModel.SMITTESTOP_NOT_ACTIVE_TEXT, 16, 22);
+            StyleUtil.InitLabel(moreInformationDescriptionLabel, StyleUtil.FontType.FontRegular,
+                SmittestopNotActiveViewModel.SMITTESTOP_NOT_ACTIVE_MORE_INFO, 18, 24);
+            StyleUtil.InitButtonWithArrowStyling(moreInformationButton, SmittestopNotActiveViewModel.SMITTESTOP_NOT_ACTIVE_INFO_LINK_TEXT);
+            StyleUtil.InitButtonStyling(languageButton, SmittestopNotActiveViewModel.SMITTESTOP_NOT_ACTIVE_BUTTONT_TEXT);
```

Comment:
I think the button should be higher and it looks like the font in the button text is wrong